sql/log.cc

```diff
@@ -709,19 +709,23 @@ bool File_query_log::open()
     goto err;
 
   {
+    char log_creation_time[iso8601_size];
+    make_iso8601_timestamp(log_creation_time);
+
     char *end;
-    size_t len=my_snprintf(buff, sizeof(buff), "%s, Version: %s (%s). "
+    size_t len=my_snprintf(buff, sizeof(buff), "%s, Version: %s (%s), Time: %s. "
 #ifdef EMBEDDED_LIBRARY
                         "embedded library\n",
-                        my_progname, server_version, MYSQL_COMPILATION_COMMENT
+                        my_progname, server_version, MYSQL_COMPILATION_COMMENT,
+                        log_creation_time
 #elif _WIN32
                         "started with:\nTCP Port: %d, Named Pipe: %s\n",
                         my_progname, server_version, MYSQL_COMPILATION_COMMENT,
-                        mysqld_port, mysqld_unix_port
+                        log_creation_time, mysqld_port, mysqld_unix_port
 #else
                         "started with:\nTcp port: %d  Unix socket: %s\n",
                         my_progname, server_version, MYSQL_COMPILATION_COMMENT,
-                        mysqld_port, mysqld_unix_port
+                        log_creation_time, mysqld_port, mysqld_unix_port
```

Comment:
can you also please paste sample log before and after your change?

Reply:
Before:
`percona-server/5.7/cmake-build-debug/sql/mysqld, Version: 5.7.33-36-debug-log (Source distribution). started with:
Tcp port: 13001  Unix socket: percona-server/5.7/cmake-build-debug/mysql-test/var/tmp/mysqld.1.sock`

After:
`percona-server/5.7/cmake-build-debug/sql/mysqld, Version: 5.7.33-36-debug-log (Source distribution), Time: 2021-07-05T19:55:56.317272Z. started with:
Tcp port: 13001  Unix socket: percona-server/5.7/cmake-build-debug/mysql-test/var/tmp/mysqld.1.sock`

Reply:
Thanks